.github/workflows/docker-publish.yml

```diff
@@ -0,0 +1,98 @@
+name: Docker
+
+# This workflow uses actions that are not certified by GitHub.
+# They are provided by a third-party and are governed by
+# separate terms of service, privacy policy, and support
+# documentation.
+
+on:
+  schedule:
+    - cron: '31 6 * * *'
```

Comment:
What is the purpose of this?

Reply:
to be honest im not sure, iirc this is meant to be a catch all template, so i'll just remove this because this is not something that needs to run scheduled

Reply:
removed